website/src/components/wordpress.tsx

```diff
@@ -44,8 +46,19 @@ export const WordpressPageContents = ({
 }: {
   content: string | null
 }) => {
+  const { "*": slug } = useRouteParams()
+
+  let parsed
+
   if (content) {
-    const parsed = parse(content, parseOptions)
```

Comment:
You can move `let parsed` into the `if` block. Plus it looks like you run the same `parse(content, parseOptions)` expression in both inner blocks. So you could do

```ts
const parsed = parse(content, parseOptions)
if (slug?.includes("lessons/")) {
  return <div ...>{parsed}</div>
} else {
  return parsed
}
```